package.json

```diff
@@ -28,10 +28,19 @@
     "grunt-sass": "^1.0.0",
     "grunt-shell": "^1.3.0",
     "node-sass": "^3.2.0",
-    "readdir": "0.0.6"
+    "readdir": "0.0.6",
+    "standard": "^7.1.2"
   },
-  "scripts": {
-    "test": "./node_modules/grunt-cli/bin/grunt test"
+  "scripts":{
+    "test": "standard && ./node_modules/grunt-cli/bin/grunt test"
+  },
+  "standard": {
+    "ignore": [
+      "lib",
+      "public/javascripts/vendor/",
+      "node_modules",
```

Comment:
I *think* you don't need "node_modules" here but having it doesn't do any harm either.